### ISSUE TITLE
issue with minified version not validating correctly

### ISSUE THREAD

**Issue by mcgoooo** [JamesEggers1/node-luhn#12]
Hello,

i was testing out [worldpay test card numbers](http://support.worldpay.com/support/kb/bg/testandgolive/tgl5103.html) and came across a strange bug that only seems to happened in the minified version of your code, for some reason the number below does not validate, i have made a codepen to illustrate
4917610000000000003
http://codepen.io/anon/pen/BzwakN?editors=1000#0

**Comment by JamesEggers1** [comment]
Thanks for pointing this out to me.  I'll get it the package updated in the near future along with other updates.

**Comment by mcgoooo** [comment]
Perfect, cheers

**Comment by maxcbc** [comment]
@JamesEggers1 This I think is because `./src/luhn-min.js` is not up to date with `./src/luhn.js`.

_prettified luhn-min.js_
```javascript
"use strict", module.exports = function() {
    function a(a) {
        var b = a.trim(),
            c = b.length,
            d = parseInt(b, 10),
            e = 0,
            f, g;
        if (c === 0) return !0;
        if (isNaN(d) || !/^[0-9]+$/.test(b)) return !1;
        for (var h = c; h > 0; h--) {
            f = Math.floor(d) % 10, e += f, h--, d /= 10, f = Math.floor(d) % 10, g = f * 2;
            switch (g) {
                case 10:
                    g = 1;
                    break;
                case 12:
                    g = 3;
                    break;
                case 14:
                    g = 5;
                    break;
                case 16:
                    g = 7;
                    break;
                case 18:
                    g = 9;
                    break;
                default:
                    g = g
            }
            d /= 10, e += g
        }
        return e % 10 === 0
    }
    return {
        validate: a
    }
}();
```

**Comment by test82987** [comment]
hi, is the promised on Jul 13, 2016 update ready yet?

**Comment by JamesEggers1** [comment]
I just uploaded a corrected minified file as version 2.2 of the npm package.
